### ISSUE TITLE
Possible inconsistent multiplicity of attributes

### ISSUE THREAD

**Issue by goto40** [textX/textX#357]
While playing with the code I found an effect concerning the multiplicity of attributes in case of multiple assignment to the same attribute (http://textx.github.io/textX/stable/grammar/#multiple-assignment-to-the-same-attribute) in one sequence or in more complicated parse tree constellations (see "OK"/"FAILED" below).

```
def test_attribute_multiplicity():
    """
    Test http://textx.github.io/textX/stable/grammar/#multiple-assignment-to-the-same-attribute
    """
    grammar = """
    Rule1: a='a' a='c';
    Rule2: (a='a'|a='b') a='c';
    """
    meta = metamodel_from_str(grammar)
    assert meta['Rule1']._tx_attrs['a'].mult == '1..*'            // OK
    assert meta['Rule2']._tx_attrs['a'].mult == '1..*'            // FAIL (result="1")
```

Is it worth to make further investigations? Or is this a border case and we can adapt the doc? Or did I misunderstood something?

**Comment by goto40** [comment]
Note: there is a branch https://github.com/textX/textX/tree/analysis/multiplicity to play with the effect.

**Comment by igordejanovic** [comment]
Good catch. It looks like a bug to me. The case is very specific but I guess it is worth further investigation. We should see if and how it can affect parsing. The interplay of OrderedChoice and Sequences together with multiple assignment can be tricky.
